### PR TITLE
PR to address #19

### PR DESCRIPTION
**Pull request by gregstarr**
This PR has changes to address #19:
* changed `helpers.checklat` so it doesn't error on NaN input (NaNs will be left in place)
* change -9999 output to NaN to be consistent across various transformations and various invalid inputs
* added tests for `convert` and `checklat` to test out NaN inputs
* fixed issue related to #28 but for apex to qd

This PR also includes a fix for setup.cfg which prevented appveyor from running pytest. The problem was that `igrf13coeffs.txt` was missing, then `apexpy.Apex()` would silently fail because the error happens in fortran and just crashes python. I also added a check in `Apex.set_epoch` to avoid this particular silent failure in the future.

### CODE REVIEW

File: tests/test_Apex.py [hunk 1]
```diff
@@ -378,6 +378,26 @@ def test_convert_invalid_transformation():
         apex_out.convert(0, 0, 'geo', 'foobar')
 
 
+coord_names = ['geo', 'apex', 'qd']
+
+
+@pytest.mark.parametrize('transform', itertools.product(coord_names,
+                                                        coord_names))
+def test_convert_withnan(transform):
+    N_nans = 5
```
Comment (aburrell) [inline tests/test_Apex.py:387]: Here, the docstring is missing and `N_nans` isn't snake case.  Again, fixing this myself.

**Review by aburrell — COMMENTED** [review]
Just a few things to fix, so I did it myself (commit forthcoming).  I ran this against the code example given by @ljlamarche and it performed as desired.

**Comment by gregstarr** [comment]
Thanks for taking a look

**Comment by ljlamarche** [comment]
Yep, this seems to work perfectly!  Nice job, thanks!

**Review by aburrell — APPROVED** [review]
Thanks for doing the heavy lifting.  This should pass now.  I just checked the coveralls failure and there are now no new uncovered lines so I am fine with the drop in coverage.

File: tests/test_Apex.py [hunk 2]
```diff
@@ -1301,6 +1319,29 @@ def test_set_epoch():
...
+    """Test raises OSError when IGRF coefficient file is missing."""
+
+    # Ensure the coefficient file exists
+    igrf_file = os.path.join(os.path.dirname(helpers.__file__),
+                             'igrf13coeffs.txt')
+    tmp_file = "temp_coeff.txt"
+    assert os.path.isfile(igrf_file)
+
+    # Move the coefficient file
+    os.rename(igrf_file, tmp_file)
```
[the remark this answers is not in the document]
Comment (aburrell) [reply tests/test_Apex.py:1331]: I agree with you about it.  Originally planned to take the last route in the future, but I like the first option best.  This makes it possible for people who want to use apexpy with custom coefficient files do so.

**Pull request merged**